megamek/unittests/megamek/client/bot/princess/BasicPathRankerTest.java

```diff
@@ -1121,79 +1148,70 @@ public void testCalculateDamagePotential() {
         final BasicPathRanker testRanker = Mockito.spy(new BasicPathRanker(mockPrincess));
         Mockito.doReturn(mockFireControl).when(testRanker).getFireControl();
 
-        final LosEffects mockLosEffects = Mockito.mock(LosEffects.class);
-        Mockito.when(mockLosEffects.canSee()).thenReturn(true);
-        Mockito.doReturn(mockLosEffects).when(testRanker).calcLosEffects(Mockito.any(IGame.class), Mockito.anyInt(),
-                                                                         Mockito.any(Targetable.class));
+        final IBoard mockBoard = generateMockBoard();

```

Comment:
I realized too late that the old calcLosEffects method must have been there for unit test purposes.

On the plus side, this improves test coverage slightly on the actual calcLosEffects method, so I think we can call it a win.